### PR TITLE
formatting reboot (space-2 and consistency)

### PR DESCRIPTION
**Pull request by rhutchison**
clean up tabs and consistently define function declarations

### CODE REVIEW

**Comment by rhutchison** [comment]
@lirantal @ilanbiala this either needs to be merged quickly or closed.

**Comment by codydaig** [comment]
@rhutchison I thought we were switching too 2 spaces instead of tabs?

**Comment by rhutchison** [comment]
Only for package.json and bower.json

**Comment by codydaig** [comment]
@rhutchison  Quoting @lirantal in #670 "Seems like there's no discussion required on the indentation type (thank god!) - we'll go with 2 spaces indent like the rest of the world"

**Comment by lirantal** [comment]
yeah we want to go for 2 spaces everywhere in the project and drop the tabs thing, although I was thinking we'll do that after the 0.4.0 release.

@ilanbiala WDYT? do you want to merge this in so that at least have tabs consistency everywhere right now?

**Comment by ilanbiala** [comment]
2 spaces please, that will also need to be changed in all our linter configs.

**Comment by rhutchison** [comment]
@ilanbiala @lirantal @trainerbill @mleanos @simison @codydaig please review.

I will add JSCS as a separate PR

**Comment by codydaig** [comment]
LGTM! So glad we're moving to spaces! :-D
Thanks @rhutchison!

**Comment by lirantal** [comment]
@rhutchison I noticed this PR deletes the file  modules/core/client/img/brand/logo.png and also other image files show up different.

**Comment by rhutchison** [comment]
@lirantal good catch please review again.  I also updated admin files since it was merged :)

**Comment by ilanbiala** [comment]
@rhutchison we should do this as the last PR for 0.4.0 so we don't need to keep redoing.

**Comment by rhutchison** [comment]
@ilanbiala we should do it ASAP so I don't have to keep fixing merge conflicts.  Everyone should update their PRs

**Comment by ilanbiala** [comment]
@rhutchison why do we have JSCS and JSHint?

**Comment by rhutchison** [comment]
@ilanbiala I don't have experience with jscs, so I wasn't sure if we viewed jscs as a supplement to/replacement of jshint.  I have noticed some projects use both - is there anything in jshint that would not be covered by jscs?

**Comment by ilanbiala** [comment]
@rhutchison I've never noticed an issue with just using JSHint, so I'm not really sure that it's needed unless someone else thinks so.

**Comment by rhutchison** [comment]
Can we keep it in for now, and decide to remove it later?

**Comment by ilanbiala** [comment]
@lirantal thoughts? I'm not really sure we should keep it unless we have a specific reason for it existing.

**Comment by lirantal** [comment]
I tend to agree with @ilanbiala , just adding it for temporary place holder until we figure out doesn't sound like a reasonable decision to make.

I also don't like that we have both jshint, and jscs, and we might be ok with replacing jshint in favor of the latter but that requires more discussion and also will not happen in grunt-life-time so it means post 0.4.0.

I say let's remove it for now and talk about it in the coding convention issue that we have open to also sort out the rules we want to follow.

File: config/config.js [hunk 1]
```diff
@@ -4,179 +4,180 @@
...
+  } else if (_.isString(globPatterns)) {
+    if (urlRegex.test(globPatterns)) {
+      output.push(globPatterns);
+    } else {
+      var files = glob.sync(globPatterns);
+      if (excludes) {
+        files = files.map(function (file) {
+          if (_.isArray(excludes)) {
+            for (var i in excludes) {
+              file = file.replace(excludes[i], '');
```
Comment (lirantal) [inline config/config.js]: what's up with this odd diff here? looks like just spacing, but nothing was actually changed in the code, right?

**Comment by lirantal** [comment]
Looks good.
Why do we need JSCS in there if there's no grunt module for that? and doesn't that conflict with anything else?

File: config/config.js (hunk 1 above)
Comment (rhutchison) [inline config/config.js]: @lirantal correct - indentation and space after anonymous functions

**Comment by lirantal** [comment]
@rhutchison let's merge this formatting PR for 0.4.0 but let's keep it indentation related so please remove JSCS for now, we can look at adding it later, squash everything and we're done.

**Comment by rhutchison** [comment]
@lirantal @ilanbiala cleaned up and ready to merge

**Comment by codydaig** [comment]
@rhutchison LGTM. Thanks for putting in the effort to submit and keep up on this PR! I'm glad to see spaces over tabs finally.

**Pull request merged**